devel/ansible-podman/containers/bodhi/bodhi.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=bodhi
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Environment=PYTHONWARNINGS=once
+WorkingDirectory=/bodhi/bodhi-server
+ExecStartPre=/usr/bin/poetry -C /bodhi/bodhi-messages install --only-root
+ExecStartPre=/usr/bin/poetry -C /bodhi/bodhi-client install --only-root
+ExecStartPre=/usr/bin/poetry -C /bodhi/bodhi-server install --only-root
```

Comment:
Poetry will install its own pinned dependencies and entirely replace the system ones installed in the Containerfile. Is this what you want?

Reply:
`--only-root` avoids that. It only "installs" (actually, it just drops a pointer to, really) the app itself, not any of the dependencies. As the `poetry install` help says: "--only-root            Exclude all dependencies."

Reply:
Ah, okay.

Reply:
for context, all the stuff to actually set up the environment inside the Bodhi container is ultimately derived from the ansible plays that do the same job for the VM env. In there we actually do run a full `poetry install`, which does exactly what you say (so it wastes a bunch of time, bloats the environment, and means we're not testing how Bodhi is actually deployed in real life).

I found that and decided to change it to use `--only-root` while I was trying to figure out why the VM env also uses a Python virtualenv. I think the virtualenv is just to make the `poetry install` not overwrite a bunch of system files or do something else weird, but...if we just make it not try and update anything, we also don't need the virtualenv...:) Took a while to unpick all of that. We *could* try not running the `poetry install` *at all*, I guess, but I'm pretty sure it'll break something.